src/sgame/sg_svcmds.cpp

```diff
@@ -269,6 +269,68 @@ static gclient_t *ClientForString( char *s )
 	return &level.clients[ idnum ];
 }
 
+/*
+===================
+Svcmd_EntityLock_f
+===================
+*/
+
+static void Svcmd_EntityLock_f()
+{
+	char      argument[ 4 ];
+	int       e;
+	gentity_t *door;
```

Comment:
it's easier to read when variables are declared as close as where they're used/init (nitpick)

Reply:
I moved `mins` and `maxs` to a smaller scope. Let's resolve this.